### ISSUE TITLE
 Can you provide it in tsx format?

### ISSUE THREAD

**Issue by star8618** [macvk/dnsleaktest#20]
Can you provide it in tsx format?

**Comment by macvk** [comment]
yes, sure. Could you please provide an example of the file you need. I've googled and can't figure out what format should I follow...
